cmd/fs-v1-metadata.go

```diff
@@ -375,6 +383,8 @@ func checkFormatSanityFSV2(fsPath string) error {
 // Check for sanity of FS format for a given version.
 func checkFormatSanityFS(fsPath string, fsFormatVersion string) (err error) {
 	switch fsFormatVersion {
+	case fsFormatV1:
```

Comment:
checkFormatSanityFS() and checkFormatSanityFSV1() can be removed. Because we would have already migrated from previous versions to the latest version by now.

and in initFormatFS() you can call checkFormatSanityFSV2() directly